src/handlers/payout/action.ts

```diff
@@ -150,7 +150,17 @@ export const handleIssueClosed = async () => {
   const shortenRecipient = shortenEthAddress(recipient, `[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]`.length);
   logger.info(`Posting a payout url to the issue, url: ${payoutUrl}`);
   const comment = `### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
-  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
+  const permitComments = comments.filter((content) => {
+    if (content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot) {
+      const url = new URL(content.body);
+      const searchParams = new URLSearchParams(url.search);
```

Comment:
URLSearchParams are already included in the URL object so no need to parse again.